samples/client/petstore/typescript-jquery/default/api/PetApi.ts

```diff
@@ -39,7 +39,7 @@ export class PetApi {
 
     private extendObj<T1, T2 extends T1>(objA: T2, objB: T2): T1|T2 {
         for (let key in objB) {
-            if (objB.hasOwnProperty(key)) {
+            if (objB[key]) {
```

Comment:
where does this change come from?

Reply:
Hmm... That should not be there, something went wrong with rebasing. Will repush.